src/CMakeLists.txt

```diff
@@ -29,24 +29,13 @@ endif()
 if (GIF_LIBRARY)
     target_link_libraries       (leptonica ${GIF_LIBRARY})
 endif()
-if (JPEG_LIBRARY)
-    target_link_libraries       (leptonica JPEG::jpeg)
-endif()
 if (JP2K_FOUND)
     target_link_libraries       (leptonica ${JP2K_LIBRARIES})
 endif()
-if (PNG_LIBRARY)
-    target_link_libraries       (leptonica PNG::png)
-endif()
-if (TIFF_LIBRARY)
-    target_link_libraries       (leptonica TIFF::libtiff)
-endif()
 if (WEBP_FOUND)
     target_link_libraries       (leptonica ${WEBP_LIBRARIES})
 endif()
-if (ZLIB_LIBRARY)
-    target_link_libraries       (leptonica ZLIB::zlib)
-endif()
+target_link_libraries           (leptonica JPEG::jpeg PNG::png TIFF::libtiff ZLIB::zlib)
```

Comment:
The example app already gets these libraries as an interface link with the target.
This didn't change when adding PUBLIC and I wouldn't expect it to.